src/test/java/com/snowflake/kafka/connector/ConnectorConfigTest.java

```diff
@@ -188,6 +188,17 @@ public void testEmptyHost() {
     }
   }
 
+  @Test
+  public void testEmptyCleanFileIfStatusLoadedCheck() {
+    try {
+      Map<String, String> config = getConfig();
+      config.remove(SnowflakeSinkConnectorConfig.SNOWPIPE_CLEAN_FILES_ONLY_IF_STATUS_LOADED);
```

Comment:
does this `config` object have `SNOWPIPE_CLEAN_FILES_ONLY_IF_STATUS_LOADED` ? 
What's getting tested in this test ? 🤔